Traceback (most recent call last):
  File "C:\projects\BattleKiller2D\main.py", line 877, in __init__
    self.s.connect((sip, sport))
ConnectionRefusedError: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение
